components/library/librsvg/Makefile

```diff
@@ -13,24 +13,30 @@
 # Copyright 2014 Alexander Pyhalov.  All rights reserved.
 #
 
-include ../../make-rules/shared-macros.mk
+include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= librsvg
-COMPONENT_VERSION= 2.36.4
+COMPONENT_MAJOR_VERSION= 2.39
+COMPONENT_VERSION= $(COMPONENT_MAJOR_VERSION).0
+COMPONENT_FMRI= image/library/librsvg
+COMPONENT_CLASSIFICATION= System/Multimedia Libraries
 COMPONENT_SUMMARY= Library for SVG support for GNOME
 COMPONENT_SRC= $(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE= $(COMPONENT_SRC).tar.xz
 COMPONENT_ARCHIVE_HASH= \
-  sha256:1021935204798f4f0ad3004a09b583668ea94a48593461b147fdcff68a18e6c2
+  sha256:aa47dcde0128eee6e3595d203bc673d9c27389588842f401bf585f31fc65095f
 COMPONENT_ARCHIVE_URL= \
-  http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/2.36/$(COMPONENT_ARCHIVE)
-COMPONENT_PROJECT_URL = https://www.gnome.org/
+  http://ftp.gnome.org/pub/GNOME/sources/$(COMPONENT_NAME)/$(COMPONENT_MAJOR_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL= https://www.gnome.org/
+COMPONENT_LICENSE= LGPLv2
 
-include ../../make-rules/prep.mk
-include ../../make-rules/configure.mk
-include ../../make-rules/ips.mk
+include $(WS_MAKE_RULES)/prep.mk
+include $(WS_MAKE_RULES)/configure.mk
+include $(WS_MAKE_RULES)/ips.mk
 
-PATH=/usr/gnu/bin:/usr/bin
+PATH=$(PATH.gnu)
+
+CPPFLAGS+= -P
```

Comment:
Good question... I need to check again in the gcc 7 zone if there was a problem. Another question is why are they not in general?